.travis.yml

```diff
@@ -1,19 +1,27 @@
 # Config file for automatic testing at travis-ci.org
-dist: trusty
+dist: focal
 
 language: python
 
 python:
+  - "3.9-dev"
+  - "3.8"
+  - "3.7"
   - "3.6"
   - "3.5"
-  - "3.4"
   - "2.7"
-#  - "pypy"
+# - "pypy3"
+
+jobs:
+  allow_failures:
+    - python: "3.9-dev"
```

Comment:
why test against this if we will allow it to fail. Maybe stopping at 3.8 is good enough for now.

Reply:
The Travis CI documentation around allow_failures are quite clear about when it should be used.

Python 3.9 releases next week.

Reply:
any further changes before i merge this PR?

Reply:
No further changes.  Thanks.